oak/server/rust/oak_runtime/src/node/wasm/mod.rs

```diff
@@ -91,9 +89,9 @@ impl WasmInterface {
     }
 
     /// Creates a new `WasmInterface` structure.
-    pub fn new(pretty_name: String, runtime: RuntimeProxy) -> WasmInterface {
+    pub fn new(pretty_name: &str, runtime: Option<RuntimeProxy>) -> WasmInterface {
```

Comment:
Can you clarify in the comment why the second arg is an `Option`?

Reply:
Went for a different approach, see below.